yugabyte/src/yugabyte/ysql/bank_improved.clj

```diff
@@ -13,8 +14,9 @@
 
 (defn- read-accounts-map
   "Read {id balance} accounts map from a unified bank table using force index flag"
-  ([op c]
-   (c/execute! c ["SET yb_read_from_followers = true"])
+  ([test op c]
+   (if (v/newer-or-equal? (:version test) "2.8.0.0-b1")
+     (c/execute! c ["SET yb_read_from_followers = true"]))
```

Comment:
This should be reworked:
1. There should be a configuration variable controlling whether we want this optimization if available
2. Another configuration var denoting the minimum version where follower reads are available
3. Actual code should use those named variables rather than magic numbers and hardcode